app/backend/templates/content_report.md

```diff
@@ -0,0 +1,39 @@
+---
+subject: "Content Report #{{ report.id|couchers_escape }}"
+---
+
+{% from "macros.html" import button, link, support_email, email_link, newline %}
+
+Content Report #{{ report.id|couchers_escape }}.
+
+
+* Reason{{ newline(html)|couchers_safe }}
```

Comment:
Trust markdown's ability to render html from text, instead of cluttering the file like this.